install.rb

```diff
@@ -299,11 +299,17 @@ def prepare_installation
   bindir = join(destdir, bindir)
   mandir = join(destdir, mandir)
   sitelibdir = join(destdir, sitelibdir)
+  vardir = "/opt/puppetlabs/agent/cache"
+  rundir = "/var/run/puppetlabs/agent"
+  logdir = "/var/log/puppetlabs/agent"
 
   FileUtils.makedirs(configdir) if InstallOptions.configs
   FileUtils.makedirs(bindir)
   FileUtils.makedirs(mandir)
   FileUtils.makedirs(sitelibdir)
+  FileUtils.makedirs(vardir)
+  FileUtils.makedirs(rundir)
+  FileUtils.makedirs(logdir)
```

Comment:
I'm not sure if this bit is the right way to get these directories. Since the new paths require us to create multiple sub-directories, it seems like it might be necessary.